lib/rules/web/admin_console/4.6/clustersettings.xyaml

```diff
@@ -30,3 +30,17 @@ click_update_channel_button:
   - selector:
       xpath: //button[@data-test-id='current-channel-update-link']
     op: click
+click_channels_drop_down:
+  elements:
+  - selector:
+      xpath: //button[@data-test-id='dropdown-button']
+    op: click
```

Comment:
same as above, we can use `click_dropdown_button` directly, no need to add a new rule

Reply:
@yapei update:
- removed the redundant rule and used the existing web rule as suggested.